### PR TITLE
Migrate pipeline resources to v1beta1

### PR DESCRIPTION
**Pull request by afrittoli**
# Changes

Migrate pipeline resources to v1beta1.
This requires pipelines v.0.11.x minimum.

Depends on #280 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [n/a ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: tekton/ci/conditions.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-apiVersion: tekton.dev/v1alpha1
+apiVersion: tekton.dev/v1beta1
```
Comment (vdemeester) [inline tekton/ci/conditions.yaml:1]: These are still `v1alpha1`

File: tekton/ci/conditions.yaml [hunk 2]
```diff
@@ -29,7 +29,7 @@ spec:
       git diff-tree --no-commit-id --name-only -r HEAD $BACK | \
           grep -E '$(params.regex)'
 ---
-apiVersion: tekton.dev/v1alpha1
+apiVersion: tekton.dev/v1beta1
```
Comment (vdemeester) [inline tekton/ci/conditions.yaml:32]: These are still `v1alpha1`

File: tekton/ci/conditions.yaml (hunk 1 above)
Comment (afrittoli) [reply tekton/ci/conditions.yaml]: done

File: tekton/ci/conditions.yaml (hunk 2 above)
Comment (afrittoli) [reply tekton/ci/conditions.yaml]: done

**Pull request merged**